src/templates/ConditionTemplate.js

```diff
@@ -0,0 +1,115 @@
+const { extension, coding, valueCodeableConcept, reference } = require('./snippets');
+const { ifAllArgsObj } = require('../helpers/templateUtils');
+
+function histologyTemplate({ histology }) {
+  return {
+    url: histology.url,
+    ...valueCodeableConcept({ ...histology }),
```

Comment:
I think if you're not passing in any additional properties you can just pass in `histology` without using the spread operator.
```suggestion
    ...valueCodeableConcept(histology),
```

I think there's several other instances with `coding` that this change can be made also in this file.

Reply:
oooh that's a good catch.